app/selenium_ui/confluence/pages/pages.py

```diff
@@ -83,6 +85,19 @@ def wait_for_comment_field(self):
     def click_edit(self):
         self.wait_until_clickable(PageLocators.edit_page_button).click()
 
+    def check_resources_loaded(self):
+        attempts = 3
```

Comment:
We do not need this variable. Need to break if True and continue iteration if not. Print message after full `for` loop.